### DISCUSSION TITLE
How do I set a border for blink.cmp?

### DISCUSSION THREAD

**Discussion by CSYYDS** [LazyVim/LazyVim#5056] [General]
I took a quick look at blink.cmp, and it doesn't seem to have a very detailed description of the configuration. I guessed and added a border to it, but its color is different from that of noice.nvim. Is there a simple way to directly configure it?

<img width="529" alt="image" src="https://github.com/user-attachments/assets/94a7f5d0-5940-4c06-aebe-8ce7475a30de" />
<img width="494" alt="image" src="https://github.com/user-attachments/assets/ee458e69-c1c4-4d76-b998-8bf04e244dc8" />

**Comment by georgeguimaraes** [comment]
i got it working with

```lua
  {
    "saghen/blink.cmp",
    opts = {
      completion = {
        menu = {
          border = "rounded",
          winhighlight = "Normal:BlinkCmpDoc,FloatBorder:BlinkCmpDocBorder,CursorLine:BlinkCmpDocCursorLine,Search:None",
        },
        documentation = {
          window = {
            border = "rounded",
          },
        },
      },
    },
  },

```

**Reply by CSYYDS** [reply]
I have set it up the same way, but there are color differences!
